docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -2,17 +2,44 @@
 
 <p class="description">This guide explains how to upgrade from Material UI v6 to v7.</p>
 
-## Start using the alpha release
+## Why you should upgrade to Material UI v7
 
-In the `package.json` file, change the package version from `latest` to `next`.
+### Improved ESM support
 
-```diff title="package.json"
--"@mui/material": "latest",
-+"@mui/material": "next",
-```
+Our package layout has been updated, we now unambiguously support both valid ESM and commonjs through the `exports` field in `package.json`.
+You can read more about this in the [Node.js documentation](https://nodejs.org/api/packages.html#packages_exports).
+
+This update fixes several issues with popular bundlers like Vite and webpack, and makes it possible to load our packages from ESM modules under Node.js.
```

Comment:
I guess I'm on the fence about referring to them as "our" packages. Also I believe "ESM modules" is redundant - like "ATM machine" 😁

```suggestion
This update fixes several issues with popular bundlers like Vite and webpack, and makes it possible to load MUI packages from ES modules under Node.js.
```